continuous-integration/check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+
+# Get and install dependencies
+echo
+echo '-- Prepare --'
+echo "Running on: $DISTRO $OS"
+cd /cwd
+
+if [[ $DISTRO == "ubuntu" ]]; then
+    apt-get update -qq
+    apt-get -y install build-essential
+    apt-get -y install libmount-dev libpcre3-dev python-dev dh-autoreconf flex bison pkg-config zlib1g-dev libffi-dev
+    apt-get -y install libxml2-utils libgamin-dev
+    apt-get -y install software-properties-common
+    add-apt-repository -y ppa:ricotz/testing
+    apt-get update
+    apt-get -y install libmozjs-31-dev
+else
+    exit 1
+fi
+
+# Build glib 2.0
```

Comment:
> Have you considered using JHbuild instead of doing this?

No; but, maybe, JHbuild is the way to go. I dislike it for 3 reasons:
- I'm not sure exactly what jhbuild is doing; It can do some tricks to build the software; 
- No tricks are allowed; Or, whatever it does is acceptable?
- I'm trying to assure the software build and run properly in anyone's (clean distro) computer; 

All distros out there uses jhbuild when packaging GNOME?